services/director-v2/src/simcore_service_director_v2/api/routes/computations.py

```diff
@@ -154,6 +156,7 @@ async def create_computation(
             state=RunningState.PUBLISHED
             if job.start_pipeline
             else RunningState.NOT_STARTED,
+            pipeline=nx.to_dict_of_lists(dag_graph),
             url=f"{request.url}/{job.project_id}",
             stop_url=f"{request.url}/{job.project_id}:stop"
```

Comment:
FYI: I know this is not from this PR but for reverse urls lookups, it is safer to use https://www.starlette.io/routing/#reverse-url-lookups

Reply:
looks good. will think about it next time cause I'm not too sure how to use it here.